### DISCUSSION TITLE
is there a way to list all league that your account participated?

### DISCUSSION THREAD

**Discussion by JunkyardCat** [cwendt94/espn-api#206] [Q&A]
like it will tell me all the league i have been a part of?

**Comment by cwendt94** [comment]
Not currently, this package focus on the league as a whole and less on the user information.

In the future I want to expose this package's internal ESPN API wrapper request class that could expand to getting user specific information.

**Reply by JunkyardCat** [reply]
is there a way to like contact espn to inquire about the espn api? i was thinking if i can ask them about the long issue we had with username and password

**Reply by cwendt94** [reply]
Unfortunately, I don't think there is a way to contact ESPN about their API. Because their API is technically private and they do not share any of their documentation/endpoints I don't believe they will provide any support for a open source package.

We figured out to support username and password that we would need to use a captcha solving service, which is not feasible for  a open source project. Thats why we will be removing the username and password feature.

**Reply by JunkyardCat** [reply]
if you dont mind me asking how did you guys able to figure out the end points and apis if espn is private about thier whole api thing

**Reply by cwendt94** [reply]
Sure so to figure out most of ESPN API's you have to look at the network traffic on their website. If you open your league in the web browser and then use the browsers developer tools to view the network calls being made by the website. From there you can see the endpoints it is calling, response from API, and some parameters being sent.
